src/auxiliary/p4est.jl

```diff
@@ -218,25 +226,25 @@ end
 
 # Load i-th element of the sc_array info.sides of the type p[48]est_iter_face_side_t
 # 2D version
-function unsafe_load_side(info::Ptr{p4est_iter_face_info_t}, i=1)
-  return unsafe_load_sc(p4est_iter_face_side_t, unsafe_load(info).sides, i)
+function unsafe_load_side(info::PointerWrapper{p4est_iter_face_info_t}, i::Integer=1)
+  return unsafe_load_sc(p4est_iter_face_side_t, info.sides, i)
 end
```

Comment:
Depending on a possible discussion on `unsafe_load_sc` above, we should keep in mind that there are more `unsafe_load_*` functions here and below.

Reply:
I also renamed the other `unsafe_load_*` to `load_pointerwrapper_*`.